test-requirements.txt

```diff
@@ -1,11 +1,16 @@
 argcomplete~=1.8
 coverage~=4.3.4
+coverage-env-plugin~=0.1
+coverage-config-reload-plugin~=0.1
 codecov~=2.0.5
 freezegun~=0.3.9
 pytest~=3.0
 pytest-cov~=2.2
 pytest-env~=0.6.0
+pytest-instafail~=0.3.0
 pytest-mock~=1.1
+pytest-reorder~=0.1.0
+pytest-reqs~=0.0.5
```

Comment:
Ah wonderful, they fixed [it](https://github.com/di/pytest-reqs/issues/9) upstream.